source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -44,193 +44,229 @@ lldb::REPLSP SwiftREPL::CreateInstance(Status &err, lldb::LanguageType language,
                                        Debugger *debugger, Target *target,
                                        const char *repl_options) {
   if (language != eLanguageTypeSwift) {
+    err.SetErrorString("can only create a REPL for Swift");
```

Comment:
This seems like a functional'ish change for swift. Is there any chance we can trigger this and add a test?

Reply:
Good question, I'll take a look! This might end up depending on the cmake change for repl_swift you're looking into.

Reply:
Ah, this is dead code. EnumerateSupportedLanguages already filters this out. I'll update the patch.

Reply:
No, it was right the way it was.  In lldb CreateInstance methods always return an empty one of the thing they were supposed to create.  That's a general pattern.  The more general support for REPL's is nascent because we only support one of them.  But we shouldn't change how CreateInstance works.